### PR TITLE
🏗 Create a GitHub issue when automated GH Actions workflow fail

### PR DESCRIPTION
**Pull request by danielrozenberg**
Not sure whether all of these make sense, most specifically I'm unfamiliar with how/when/whom triggers publish-npm-packages.yml

### CODE REVIEW

File: .github/workflows/cut-nightly.yml [hunk 1]
```diff
@@ -35,3 +35,10 @@ jobs:
         run: node --unhandled-rejections=strict build-system/release-workflows/cut-nightly.js
         env:
           GITHUB_TOKEN: ${{ secrets.AMPPROJECTBOT }}
+
+  create-issue-on-error:
+    needs: cut-nightly
+    if: failure()
+    uses: ampproject/amphtml/.github/workflows/create-issue-on-error.yml@main
```
Comment (estherkim) [inline .github/workflows/cut-nightly.yml:42]: Does this exist?

File: .github/workflows/sweep-experiments.yml [hunk 2]
```diff
@@ -77,3 +77,10 @@ jobs:
             ```
             git checkout -t upstream/${{ steps.sweep.outputs.branch }}
             ```
+
+  create-issue-on-error:
+    needs: sweep-experiments
+    if: failure()
+    uses: ampproject/amphtml/.github/workflows/create-issue-on-error.yml@main
+    with:
+      mention: '@ampproject/build-on-duty'
```
Comment (estherkim) [inline .github/workflows/sweep-experiments.yml:86]: Similar to my past comment, don't think build on duty has enough context

File: .github/workflows/cut-nightly.yml (hunk 1 above)
Comment (danielrozenberg) [reply .github/workflows/cut-nightly.yml:42]: Aahahhahahaaaahahaha `git commit --all` doesn't add new filesssssssssss
(fixed)

File: .github/workflows/sweep-experiments.yml (hunk 2 above)
Comment (danielrozenberg) [reply .github/workflows/sweep-experiments.yml:86]: wg-infra then?

**Comment by danielrozenberg** [comment]
Bump

**Review by estherkim — APPROVED** [review]

File: .github/workflows/sweep-experiments.yml (hunk 2 above)
Comment (danielrozenberg) [reply .github/workflows/sweep-experiments.yml:92]: Switched to wg-infra 🙃

**Pull request merged**